instance/logging.py

```diff
@@ -72,14 +74,27 @@ def emit(self, record):
             object_id = obj.pk
 
         try:
-            apps.get_model('instance', 'LogEntry').objects.create(
+            log_entry = apps.get_model('instance', 'LogEntry').objects.create(
                 level=record.levelname, text=self.format(record), content_type=content_type, object_id=object_id
             )
         except ProgrammingError:
             # This can occur if django tries to log something before migrations have created the log table.
             # Make sure that is actually what happened:
             assert 'instance_logentry' not in connection.introspection.table_names()
 
+        # Send notice of entries related to any resource. Skip generic log entries that occur
+        # in debug mode, like "GET /static/img/favicon/favicon-96x96.png":
+        if content_type:
+            log_event = {
+                'type': 'object_log_line',
+                'log_entry': LogEntrySerializer(log_entry).data
+            }
+            if hasattr(obj, 'event_context'):
+                log_event.update(obj.event_context)
+            # TODO: Filter out log entries for which the user doesn't have view rights
+            # TODO: More targetted events - only emit events for what the user is looking at
```

Comment:
This means the logged events will be broadcast to everyone online in OCIM, irrespective of which org the user belongs to, right? I'll have to check some logs, but I'm looking for any security/privacy risks here. What do you think?

Reply:
@viadanna, this code along with the comments has been copied from the reverted code. This issue has been present in the previous implementation as well and since it will require more time to fix it in this ticket, it might be better to create a follow-up task for fixing this.

Reply:
Sure :+1: